runtime/queries/gn/highlights.scm

```diff
@@ -0,0 +1,47 @@
+; Copyright (C) 2021 Will Cassella (github@willcassella.com)
+; 
+; Licensed under the Apache License, Version 2.0 (the "License");
+; you may not use this file except in compliance with the License.
+; You may obtain a copy of the License at
+; 
+;         http://www.apache.org/licenses/LICENSE-2.0
+; 
+; Unless required by applicable law or agreed to in writing, software
+; distributed under the License is distributed on an "AS IS" BASIS,
+; WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+; See the License for the specific language governing permissions and
+; limitations under the License.
+
+(identifier) @variable.builtin
+;((identifier) @variable (#is? @variable definition)) ; This doesn't work
```

Comment:
I believe `#is?` is something nvim specific, this can probably be removed

Reply:
Removed